app/controllers/medications_controller.rb

```diff
@@ -88,26 +88,22 @@ def destroy
 
   # Save refill date to Google calendar
   def save_refill_to_google_calendar(medication)
-    if current_user.google_oauth2_enabled? && medication.add_to_google_cal
-      summary = "Refill for " + medication.name
-      date = medication.refill
-      CalendarUploader.new(summary: summary, date: date, access_token: current_user.token, email: current_user.email).upload_event
-    end
+    return unless current_user.google_oauth2_enabled? && medication.add_to_google_cal
+
+    summary = 'Refill for ' + medication.name
```

Comment:
100%, yes
That is something I'd like to see in a future PR but I was worried about this one being too big already, being 68 commits and all.